### PR TITLE
Adds Issue Template

### PR DESCRIPTION
**Pull request by himanshuc3**
Closes  part of #1560

### CODE REVIEW

File: openlibrary/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+### Browser, version, and operating system
+
+> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!
+
+
+****
+
+## Thank you!
```
Comment (tabshaikh) [inline openlibrary/issue_template.md]: We could remove this

File: openlibrary/issue_template.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Your help makes internetarchive better! We *deeply* appreciate your helping refine and improve this site.
```
Comment (tabshaikh) [inline openlibrary/issue_template.md]: and this too

File: openlibrary/issue_template.md (hunk 1 above)
Comment (tabshaikh) [reply openlibrary/issue_template.md]: Just keep a single line spacing too

**Comment by tabshaikh** [comment]
We aim to have an issue template which is functional and gives us the necessary information to duplicate the issue and debug it :)

**Comment by himanshuc3** [comment]
Is this enough for issue template. Do suggest if something else to be added:

### Please Describe the problem (or idea)

> What happened just before the problem occurred? Or what problem could this idea solve?


> What did you expect to see that you didn't?


### Please show us where to look

```https://openlibrary.org/ ...```


### Browser, version, and operating system

> Many bugs are related to these -- please help us track it down and reproduce what you're seeing!

**Comment by tabshaikh** [comment]
I think this is ready for merging

File: openlibrary/issue_template.md (hunk 1 above)
Comment (mekarpeles) [reply openlibrary/issue_template.md]: I agree with @tabshaikh, let's please remove the Thank text here and below

**Comment by mekarpeles** [comment]
@himanshuc3 do you have any time this week to make the small changes above? We'd love to merge this! :)

**Comment by himanshuc3** [comment]
Sure. I'd make em right away.

**Pull request merged**